README.md

```diff
@@ -49,6 +49,6 @@ Reporting an issue or a feature request
 Issues and feature requests are tracked in the [Github issue tracker](https://github.com/FriendsOfSymfony/FOSUserBundle/issues).
 
 When reporting a bug, it may be a good idea to reproduce it in a basic project
-built using the [Symfony Standard Edition](https://github.com/symfony/symfony-standard)
-to allow developers of the bundle to reproduce the issue by simply cloning it
+built using the [Symfony Standard Edition](https://github.com/symfony/symfony-standard).
+This allows developers of the bundle to reproduce the issue by simply cloning it,
```

Comment:
these are not independent clauses either. It would be `to reproduce it by simply (cloning it and following some steps)` if I use braces to display some grouping.